app/schema/widgets/textarea_widget.py

```diff
@@ -16,7 +16,7 @@ def render(self, state):
                 'label': state.schema_item.label,
                 'value': state.input or '',
                 'placeholder': '',
-                'maxChars': 2000,
+                'maxChars': state.schema_item.max_chars or 2000,
```

Comment:
We seem to still be using magic numbers, we need to get away from the 2000 and stick it as a default higher up